### PR TITLE
kubernetes and general refactor

### PR DESCRIPTION
**Pull request by RamonSaboya**
This PR does a lot of cleaning up, including:
- using kubernetes for production
- change every remaining occurrence of `server` to `api`
- separate renderer from api
- using service storage
- remove some unused files
- reorganize ports and docker-compose in general
- some small teaks

### CODE REVIEW

File: vitaes-renderer/Common.py [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
 def refresh_render_map():
-    db = pymongo.MongoClient('mongodb://root:vitaes@mongo', 27017).vitaes
-    client = db.template_info
-    for model in client.find():
-        data = model.copy()
-        del data['_id']
-        render_map[data['name']] = data
+    for filename in glob.glob('Templates/*.json'):
+        with open(filename) as json_file:
+            data = json.load(json_file)
```
Comment (Arthurlpgc) [inline vitaes-renderer/Common.py]: loads?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,32 +1,31 @@
 version: '3'
 services:
   webapp:
```
Comment (Arthurlpgc) [inline docker-compose.yml]: why isnt command here?

File: vitaes-webapp/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,9 @@
 FROM node:11.14-slim as react-build
 WORKDIR /project
-RUN yarn global add serve
 COPY package.json yarn.lock ./
 RUN yarn
 COPY . .
+RUN yarn build
+
+FROM nginx:alpine
+COPY --from=react-build /project/build/ /usr/share/nginx/html
```
Comment (Arthurlpgc) [inline vitaes-webapp/Dockerfile]: fair

File: vitaes-renderer/Common.py (hunk 1 above)
Comment (RamonSaboya) [reply vitaes-renderer/Common.py]: nops, it's load 'cause json_file is not a string

File: vitaes-logger/main.go [hunk 4]
```diff
@@ -93,5 +93,5 @@ func main() {
 		handler(w, r, db)
 	}))
 	http.Handle("/", handler)
-	log.Fatal(http.ListenAndServe(":8017", nil))
+	log.Fatal(http.ListenAndServe(":6000", nil))
```
Comment (Arthurlpgc) [inline vitaes-logger/main.go]: 6000 seems arbitrary

Comment (RamonSaboya) [reply vitaes-logger/main.go]: it is

File: docker-compose.yml (hunk 2 above)
Comment (RamonSaboya) [reply docker-compose.yml]: It's on the Dockerfile.dev

Comment (Arthurlpgc) [reply docker-compose.yml]: perfect

File: vitaes-renderer/Common.py (hunk 1 above)
Comment (Arthurlpgc) [reply vitaes-renderer/Common.py]: my bad

**Comment by Arthurlpgc** [comment]
> This PR does a lot of cleaning up, including:
> 
> * using kubernetes for production
> * change every remaining occurrence of `server` to `api`
> * separate renderer from api
> * using service storage
> * remove some unused files
> * reorganize ports and docker-compose in general
> * some small teaks

Better name for the PR

**Pull request merged**